lib/datadog/core/utils/compression.rb

```diff
@@ -6,10 +6,13 @@
 module Datadog
   module Core
     module Utils
-      # Common database-related utility functions.
+      # Compression/decompression utility functions.
+      #
+      # @deprecated This is no longer used by ddtrace and will be removed in 2.0.
```

Comment:
Does it make sense to point to the new Compression replacement, or is there none?

Reply:
There isn't any, since libdatadog ships the lz4 compression inside itself and does not (currently) expose it to callers. (Which is why I needed to get the `extlz4` gem for the rspec testing)